.research/audit_site.py

```
@@ -20,6 +20,27 @@ def rel_exists(page, href):
     target = os.path.normpath(os.path.join(os.path.dirname(page), href))
     return os.path.exists(target)
 
+# Anchor targets per page. rel_exists() throws the #fragment away before it checks anything, so a
+# link to a real page with a nonexistent heading always passed. That hid 5 dead anchors in the
+# mirrored README/STATE pages, where GitHub's "a--b" slugs do not match ours.
+ANCHORS = {}
+for _p in htmls:
+    _t = open(_p, encoding='utf-8', errors='replace').read()
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)